### PR TITLE
Added Update node tag UI modal and cypress testcases

### PR DESCRIPTION
**Pull request by himanshi-chhabra**
Signed-off-by: Himanshi Chhabra <hchhabra@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
- Add Update node tag UI on node list dropdown page.

### :chains: Related Resources
#4916 
### :+1: Definition of Done
- Added dropdown for update node tag on node list page.
### :athletic_shoe: How to Build and Test the Change
**PFA Video**
```
STEP 1
inside the hab studio

[default:/src:0]# build components/automate-ui-devproxy/
[default:/src:0]# start_automate_ui_background
[default:/src:0]# start_all_services

STEP 2
open new window
go to automate UI path

$ cd components/automate-ui
and run the command 

npm run serve:hab
```
To add data https://github.com/chef/automate/blob/master/dev-docs/adding-data/adding_test_data.md#adding-data-to-infra-views

Steps node dropdown for update tag: 
1. Go To  Infrastructure -> Chef Servers -> click server name-> click on org name -> Nodes.
2.  Click on  `...` on particular node -> click ```Manage node tag``` -> add tags in text field and click  update.

**For cypress Build**
```https://github.com/chef/automate/tree/master/e2e```
### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable

https://user-images.githubusercontent.com/61003053/119679730-375d5980-be5e-11eb-9592-c1c08106c782.mp4

### CODE REVIEW

**Review by vinay033 — APPROVED** [review]
LGTM 🎉

File: components/automate-ui/src/app/modules/infra-proxy/infra-nodes/infra-nodes.component.scss [hunk 1]
```diff
@@ -110,8 +110,15 @@ img {
 
 ::ng-deep
 .mat-select-panel .mat-option {
+
+  &.primary {
+    width: 158;
```
Comment (chaitali-mane) [inline components/automate-ui/src/app/modules/infra-proxy/infra-nodes/infra-nodes.component.scss:115]: Unit for width is missing

**Comment by himanshi-chhabra** [comment]
@chaitali-mane I have updated the changes.

**Comment by chaitali-mane** [comment]
Hi @himanshi-chhabra ,
1. When I add test, it comes as Test and 
2. When I tried with Test, It also comes as Test
Is that valid to make Capitalise Tag name? Please check .
Node-update-tag3

**Comment by chaitali-mane** [comment]
@himanshi-chhabra,
We are capitalising node name here also, Node name is case sensitive or not ? 
I found same thing on [#5086](https://github.com/chef/automate/pull/5086)
Can you please verify this

node-update-tag2

**Comment by himanshi-chhabra** [comment]
> Hi @himanshi-chhabra ,
> 
> 1. When I add test, it comes as Test and
> 2. When I tried with Test, It also comes as Test
>    Is that valid to make Capitalise Tag name? Please check .
>    Node-update-tag3

Yes, I have checked it's in the UX design to  capitalize tag name

**Comment by himanshi-chhabra** [comment]
> @himanshi-chhabra,
> We are capitalising node name here also, Node name is case sensitive or not ? Can you please verify this
> I found same thing on [#5086](https://github.com/chef/automate/pull/5086)
> 
> node-update-tag2

I have updated the changes

**Review by chaitali-mane — APPROVED** [review]
Thanks @himanshi-chhabra , Looks good to me

**Pull request merged**